### ISSUE TITLE
Error: Cannot find module 'babel-polyfill'

### ISSUE THREAD

**Issue by OlivierAlbertini** [ptmt/react-native-desktop#51]
Hi,

I have the following error when I launche examples :1234:  

```
~/Documents/ReactNative/react-native-desktop-master/packager ~
module.js:340
    throw err;
    ^

Error: Cannot find module 'babel-polyfill'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/Olivier/Documents/ReactNative/react-native-desktop-master/packager/babelRegisterOnly.js:11:1)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
~
Process terminated. Press <enter> to close the window


```

**Comment by adamtwiss** [comment]
I had a similar error. Manually installing (npm install) of babel-polyfill and babel-reigister fixed it for me.

**Comment by ptmt** [comment]
Interesting, could not reproduce even on the clean install, anyway, I'll upgrade all these dependencies when will sync with RN upstream.

**Comment by ptmt** [comment]
Fixed (I hope) in 0.6.0. Would you mind to check it again if you would have enough time?
